src/kotlin/template.handlebars

```diff
@@ -48,13 +48,14 @@ class {{{apiClassName}}} : GBIORxWebServices() {
         @Streaming
         {{/if}}
         @JvmSuppressWildcards
-        fun {{{name}}}(
-            @HeaderMap headerMap: Map<String, String>{{#if params.length}},{{/if}}
+        {{#if streaming}}{{else}}suspend {{/if}}fun {{{name}}}(
```

Comment:
@noterB For education purpose, where were there variables (say, `streaming`) passed from? Thanks.